### ISSUE TITLE
[req] elitetorrent-biz

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#9873]
empty query
```
10-17 11:56:12 Error Error downloading. indexer: elitetorrent-biz path: https://www.elitetorrent.com/peliculas/ben-10-versus-el-universo/
System.Exception: Download selector a[href^="magnet:?"] didn't match
   at Jackett.Common.Indexers.CardigannIndexer.Download(Uri link) in /home/repositories/Jackett/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 1817
   at Jackett.Server.Controllers.DownloadController.Download(String indexerID, String path, String jackett_apikey, String file) in /home/repositories/Jackett/src/Jackett.Server/Controllers/DownloadController.cs:line 56 
```

**Comment by garfield69** [comment]
the site no longer has a magnet href in the details page.
can you work out how the magnet is fetched?
https://www.elitetorrent.com/peliculas/la-caza-3/

**Comment by ilike2burnthing** [comment]
Seems to be in a debugger VM script which also clears the console every 150ms:
```
setInterval(function(){console.clear();},150);if($("#elitetorrent_com_torrent").length){$("#elitetorrent_com_torrent").click(function(event){event.preventDefault();window.open('/wp-content/uploads/files/1602723150-tt2106476 720 es.torrent');});}if($("#elitetorrent_com_magnet").length){$("#elitetorrent_com_magnet").click(function(event){event.preventDefault();window.open('magnet:?xt=urn:btih:b03c36a35bb1be32719a8b7dc2527e2d32be2c5b&amp;dn=tt2106476+720+es&amp;xl=1749177940&amp;tr=udp://tracker.openbittorrent.com:80/announce&amp;tr=udp://tracker.opentrackr.org:1337/announce');})};if(document.location.host.replace('www.','')!==window.atob('ZWxpdGV0b3JyZW50LmNvbQ')){window.location.href='https://'+window.atob('ZWxpdGV0b3JyZW50LmNvbQ');}
```
Or unminified:
```
setInterval(function() {
    console.clear();
}, 150);
if ($("#elitetorrent_com_torrent").length) {
    $("#elitetorrent_com_torrent").click(function(event) {
        event.preventDefault();
        window.open('/wp-content/uploads/files/1602723150-tt2106476 720 es.torrent');
    });
}
if ($("#elitetorrent_com_magnet").length) {
    $("#elitetorrent_com_magnet").click(function(event) {
        event.preventDefault();
        window.open('magnet:?xt=urn:btih:b03c36a35bb1be32719a8b7dc2527e2d32be2c5b&amp;dn=tt2106476+720+es&amp;xl=1749177940&amp;tr=udp://tracker.openbittorrent.com:80/announce&amp;tr=udp://tracker.opentrackr.org:1337/announce');
    })
}
;if (document.location.host.replace('www.', '') !== window.atob('ZWxpdGV0b3JyZW50LmNvbQ')) {
    window.location.href = 'https://' + window.atob('ZWxpdGV0b3JyZW50LmNvbQ');
}
```

**Comment by garfield69** [comment]
if the site now required JS to be running to generate the .torrent or magnet then this indexer will need to be converted to C#

**Comment by ngosang** [comment]
@ilike2burnthing we need to execute the JS code (eval) to get the real JS code. I think this requires a lot of effort in C#. If we can't find a better way I think we should remove this indexer.

**Comment by ilike2burnthing** [comment]
Site is currently timing out with a 522 error. Who knows, maybe it will come back with their magnet links being usable again...

**Comment by ilike2burnthing** [comment]
**TL;DR - I thought I was onto something, but the more I looked, the more problems and exceptions I found. Unless someone else can get the link more easily, it's not workable.**

-------

The only other way I can see is by using their format for naming their torrent files, but it's not completely consistent, and not all of the information if readily available.

e.g. - `https://www.elitetorrent.com/wp-content/uploads/files/1602723150-tt2106476%20720%20es.torrent`

`https://www.elitetorrent.com/wp-content/uploads/files/` is always the same.

The random number at the start is nearly the same as for the image, but minus 1 or 2 - `/wp-content/uploads/portadas/elitetorrent.com-la-caza-1602723152.jpg`

Next we have a hyphen and the IMDB ID - `tt2106476`, but this isn't present in the HTML.

Then there is a space, followed by the quality `1080` `720` `hdrip` `dvdrip`, then another space, and a 2 letter language code - `es` for Spanish, `vo` for original, `la` for Spanish Latino.

-------

But of course to make things more interesting, their microHD releases throw that out the window and use a similar structure to the page URL (kinda...):
https://www.elitetorrent.com/peliculas/una-historia-de-venganza-microhd-2-bittorrent-hd-1/
https://www.elitetorrent.com/wp-content/uploads/files/una-historia-de-venganza-microhd-2.torrent

Titles which include accents get changed, e.g. `savva. el corazón del guerrero`:
https://www.elitetorrent.com/peliculas/savva-el-corazon-del-guerrero-microhd-bittorrent-hd-1/
https://www.elitetorrent.com/wp-content/uploads/files/savva-el-coraza-sup3-n-del-guerrero-microhd.torrent

Or `manual de un tacaño`:
https://www.elitetorrent.com/peliculas/manual-de-un-tacano-microhd/
https://www.elitetorrent.com/wp-content/uploads/files/manual-de-un-tacaa-plusmn-o-microhd.torrent

And single word titles differ too, just for fun, e.g. `fences`:
https://www.elitetorrent.com/peliculas/fences/
https://www.elitetorrent.com/wp-content/uploads/files/fences-P3j5f.torrent
`P3j5f` is found nowhere in the HTML.

**Comment by ngosang** [comment]
They did a good job hiding the URLS. We should remove this tracker because. There are better alternatives for Spanish speakers out there.

**Comment by kaizokumcc** [comment]
> 
> 
> the site no longer has a magnet href in the details page.
> can you work out how the magnet is fetched?
> https://www.elitetorrent.com/peliculas/la-caza-3/

Hi, I would like to open this topic again, as I think that currently you can retrieve their magnet link directly from their source code: 

https://www.elitetorrent.com/peliculas/la-caza-3/

`<div class="ficha_descarga_opciones">
				<div class="enlace_descarga" style="padding:25px 0px;">
					<a href="/wp-content/uploads/files/1602723150-tt2106476 720 es.torrent" rel="nofollow" class="enlace_torrent degradado1">Descargar el .torrent</a>
					<a href="magnet:?xt=urn:btih:b03c36a35bb1be32719a8b7dc2527e2d32be2c5b&amp;dn=tt2106476+720+es&amp;xl=1749177940&amp;tr=udp://tracker.openbittorrent.com:80/announce&amp;tr=udp://tracker.opentrackr.org:1337/announce" rel="nofollow" class="enlace_torrent degradado1">Descargar por magnet link</a><br>
					</center>
					</div>
				
				</div>`

That way, probably with your old system, that performs a search for the magnet in the source (https://github.com/Jackett/Jackett/commit/a7b7ca658fb97a7dc551bcb31e8856e130f6aaf4) the tracker can be added again. 

I have prefered reopening this issue instead of opening a new one, let me know if it is the correct move, and thank you so much for your project!
Cheers from a spanish speaker

**Comment by garfield69** [comment]
v0.18.383

**Comment by garfield69** [comment]
https://www.elitetorrent.com/ returns DNS_PROBE_FINISHED_NXDOMAIN (DNS lookup error)

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
v0.20.2509
